gcp_variant_transforms/testing/data/input_files/combine

```diff
@@ -0,0 +1 @@
+gs://gcp-variant-transforms-testfiles/small_tests/valid-4.0*.vcf
```

Comment:
It might be worthy more to have two file_patterns for the testing purpose.

Reply:
It does - there is * which encapsulates .../valid-4.0.vcf, .../valid-4.0-one-more-filed.vcf and .../valid-4.0-new.vcf.

Reply:
Sorry, what I mean is that we provide two patterns in this file, e.g., 
`gs://gcp-variant-transforms-testfiles/small_tests/valid-4.0*.vcf`
`gs://gcp-variant-transforms-testfiles/small_tests/valid-4.1*.vcf`

The new flag essentially allows the user to provide more than one pattern, while in this test case the user can actually use `input_pattern=gs://gcp-variant-transforms-testfiles/small_tests/valid-4.0*.vcf`

Reply:
Done.